### ISSUE TITLE
improve search view

### ISSUE THREAD

**Issue by Heklaterriol** [jemproject/JEM-Project#1797]
I noticed that the JEM search view looks quite ugly and old.
I made some imrpvements.

**Comment by Heklaterriol** [comment]
before:
<img width="986" alt="search-old" src="https://github.com/user-attachments/assets/5b7f7ef7-a78a-4764-a3d5-70c7b2264668">



after:
<img width="986" alt="search-new" src="https://github.com/user-attachments/assets/12496a76-6243-49cc-ba5f-95e7e2b32100">

**Comment by mckillo** [comment]
Good style, and I have a proposal to the composition (small with 2 lines):

![image](https://github.com/user-attachments/assets/e8d72d57-13ba-4e1f-ab2a-a58c1ed68bcc)

What is your opinion? 
Very tight?

**Comment by Heklaterriol** [comment]
A comment, even if the issue is closed:
mckilllos suggestion is good, but there are two more fields appearing (country, city), after a continent is chosen.
Unfortunatey the two fields are not displayed directly ofter chossing a continent, but only after hitting the search button.
We missed this functionality when moving from mootools to jquery.

I will open a new issue for that.
